build/integrationtests/Dockerfile

```diff
@@ -1,23 +1,29 @@
 ARG cycle
 ARG hub
+ARG integrationtests
```

Comment:
If I remember correctly, all the args defined before the "FROM" clause are wiped out just after it.

Also, it is best if you use the ARGS close to where you need it which, in this case, is just after line 13... 

This makes sure the layers are better reused when you change their values...

Reply:
Done